tests/kernel/install_ltp.pm

```diff
@@ -22,21 +22,35 @@ sub add_repos {
     zypper_call('--gpg-auto-import-keys ref');
 }
 
-sub try_add_workstation_addon {
-    my $have_wse = get_var('BUILD_WE') && get_var('ISO_2');
-    if ($have_wse) {
+sub scc_we_enabled {
+    script_run("zypper -n products -i", 0);
+    return wait_serial(qr/Workstation Extension/);
```

Comment:
unfortuanetly default value is safe , calling `zypper products` refreshes repositories and services which takes time and 5 sec is too short